Sorstr.py

```diff
@@ -1,55 +0,0 @@
-from glob import glob
-
-import re
-
-
-class Sorstr:
-    inverted_index = {}
```

Comment:
This dictionary instance is shared among all instances of `Sorstr`:

```
sorstr = Sorstr()
sorstr.inverted_index['test'] = True
other = Sorstr()
print(other.inverted_index['test']) # => True
```

While this is not an issue with the existing logic, it might turn into a problem in the future.

The preferable solution is to initialize the property in the constructor.